cmd/gorepomod/internal/git/runner.go

```diff
@@ -165,8 +167,20 @@ func (gr *Runner) LoadLocalTags() (result misc.VersionMap, err error) {
 
 func (gr *Runner) LoadRemoteTags(
 	remote misc.TrackedRepo) (result misc.VersionMap, err error) {
-	gr.comment("loading remote tags")
 	var out string
+
+	// Update latest tags from upstream
+	gr.comment("updating tags from upstream")
+	_, err = gr.run(noHarmDone, "fetch", "-t", string(remoteUpstream), "master")
```

Comment:
I think the `master` branch name maybe rename, I suggest using const variable to define the upstream default branch name.
https://github.com/kubernetes-sigs/kustomize/issues/3954